system-helper/flatpak-system-helper.c

```diff
@@ -298,6 +298,8 @@ handle_deploy (FlatpakSystemHelper   *object,
     }
   else if (local_pull)
     {
+      const OstreeRepoPullFlags pull_flags = OSTREE_REPO_PULL_FLAGS_UNTRUSTED |
+                                             OSTREE_REPO_PULL_FLAGS_BAREUSERONLY_FILES;
```

Comment:
I don't think this one is needed. The call to flatpak_dir_pull() will eventually call repo_pull_one_dir, which sets this.